apis/nucleus/src/utils/style/shadow-utils.js

```diff
@@ -19,6 +19,7 @@ export const getShadows = (shadowString) => {
 
 /**
  * Combine box shadow with box shadow color.
+ * @ignore
```

Comment:
I would go with
```suggestion
 * @private
```
from what is know that will also exclude it from the spec